spec/cli/nextify_spec.rb

```diff
@@ -329,4 +329,20 @@
       output.should include('method-reference ("Language.:transform")')
     end
   end
+
+  context "with --overwrite" do
+    after do
+      old_content = File.read(File.join(__dir__, "dummy", "transpile_me.rb"))
+      File.write(File.join(__dir__, "dummy", "overwrite", "transpile_me.rb"), old_content)
+    end
+
+    it "overwrites original file if --single-version is provided" do
+      run_ruby_next(
+        "nextify #{File.join(__dir__, "dummy", "overwrite", "transpile_me.rb")} " \
+      "--single-version --overwrite"
+      ) do |_status, _output, err|
+        File.read(File.join(__dir__, "dummy", "overwrite", "transpile_me.rb")).should include("using RubyNext")
```

Comment:
Checking for `using RubyNext` fails in old Rubies (where we do not support Refinements).

We can check:
- Some other transpilation-specific expression (`__matchee__`?)—for this to work, we need to specify the `--min-version` to some version without pattern matching
- Compare original and transpiled contents (they should be different)
- Try to execute the resulting file with the current Ruby version (it should not fail)—that's the best test we can write
- All of the above 🙂

Reply:
I added checking for missing --single-version and show error
Also, fixed spec - check for --single-version missing, comparing original_file and overwritten file contents and run ruby file